xml/System.Runtime.Loader/AssemblyLoadContext.xml

```diff
@@ -189,10 +242,37 @@
         <Parameter Name="assemblyName" Type="System.Reflection.AssemblyName" />
       </Parameters>
       <Docs>
-        <param name="assemblyName">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="assemblyName">The object that describes the assembly to be loaded.</param>
+        <summary>Resolves and loads an assembly given its <see cref="T:System.Reflection.AssemblyName" />.</summary>
+        <returns>The loaded assembly.</returns>
```

Comment:
Probably should add that the method never returns null.

Reply:
adding ", or throws"